aiida/manage/tests/pytest_fixtures.py

```diff
@@ -424,8 +425,11 @@ def test_1(aiida_localhost):
     :return: The computer node
     :rtype: :py:class:`~aiida.orm.Computer`
     """
-    from aiida.common.exceptions import NotExistent
-    from aiida.orm import Computer
+    warn_deprecation(
```

Comment:
Do we really need to deprecate this?

While deprecations do not break code, they still generate work for plugin authors.
I think keeping `aiida_localhost` as shortcut for `aiida_computer_localhost()` should be innocent enough?

Reply:
No, very good point. I will keep it and simply have it use `aiida_computer_localhost`.